### PR TITLE
sshuttle 1.0.1

### PR DESCRIPTION
**Pull request by BenjaminHCCarr**
Created with `brew bump-formula-pr`.

### CODE REVIEW

File: Formula/sshuttle.rb [hunk 1]
```diff
@@ -4,9 +4,8 @@ class Sshuttle < Formula
   desc "Proxy server that works as a poor man's VPN"
   homepage "https://github.com/sshuttle/sshuttle"
   url "https://github.com/sshuttle/sshuttle.git",
-      :tag      => "v0.78.5",
-      :revision => "752a95310198886515577463a4a7e36d7f218018"
-  revision 2
+      :tag      => "1.0.1",
```
Comment (issyl0) [inline Formula/sshuttle.rb:7]:
```suggestion
      :tag      => "v1.0.1",
```

**Review by issyl0 — CHANGES_REQUESTED** [review]
Thanks for this PR. Looks like `brew bump-formula-pr` missed off the `v` somehow on the tag.

**Comment by issyl0** [comment]
Thanks for the update. Would you be able to squash the commits into one (`sshuttle 1.0.1`), please, then this will be mergeable.

**Comment by BenjaminHCCarr** [comment]
squashed

**Review by issyl0 — APPROVED** [review]
Thank you!

**Pull request closed without merging**